README.md

```diff
@@ -219,8 +219,27 @@ add your name and email address to the
 under the "Contributors" section using the format:
 `First Last <email@example.com>`.
 
+For first timer contributors to a Go project, there is how to get the project setup.
```

Comment:
`To set up the project for local development, ensure that the $GOPATH variable…`

I'm not a first-time contributor, and I had issues with this too :)